src/providers/storage/azureBlobStorage.ts

```diff
@@ -138,13 +141,41 @@ export class AzureCloudStorageService implements IStorageProvider {
             path = path ? [this.options.containerName, path].join("/") : this.options.containerName;
         }
         const files = await this.listFiles(path);
-        return files.map((blobPath) => AssetService.createAssetFromFilePath(blobPath));
+        const result: IAsset[] = [];
+        for (const key of Object.keys(files.entries)) {
+            const url = this.getUrl(files.entries[key].name);
+            const asset = AssetService.createAssetFromFilePath(url);
+            if (asset.type === AssetType.Image || asset.type === AssetType.Video) {
```

Comment:
I would consider inverting this to check whether or not it is unknown which is the 3rd option in that enum.

Reply:
Done